src/wallet/scriptpubkeyman.cpp

```diff
@@ -1964,6 +1964,13 @@ std::optional<MigrationData> LegacyScriptPubKeyMan::MigrateToDescriptor()
     return out;
 }
 
+bool LegacyScriptPubKeyMan::DeleteRecords()
```

Comment:
c50135edb6414fa5fd598f65e13eaa0e639573b9 : `DeleteLegacyRecords` (since we keep plenty of other stuff)

Reply:
We delete all of the `LegacyScriptPubKeyMan` records (which this function is a member of). There are no remaining LegacySPKM records after this.